docs/install.md

```diff
@@ -0,0 +1,88 @@
+# Installation
+
+This package provides both a C++ evaluator with python bindings and
+a set of python tools to convert, validate, and inspect files with corrections
+It is installed as a python package (see e.g.
+[this tutorial](https://packaging.python.org/tutorials/installing-packages/)
+for an introduction), using [scikit-build](https://scikit-build.readthedocs.io/)
+and thus [CMake](https://cmake.org/), for the C++ components.
+Builds have been tested in Windows, OS X, and Linux, and the python bindings
+can be compiled against both python2 (with limited functionality) and python3,
+as well as from within a CMSSW environment.
+Note that CMSSW 11_2_X and above has ROOT accessible from python 3.
+
+## Inside CMSSW
+
+For CMSSW releases from the `11_3_X` branch on, correctionlib is included
+as the `py3-correctionlib` tool, so no installation is needed.
+
+For older release cycles the following recipe can be used
+to install only the `correctionlib._core` evaluator,
+which allows to use the C++ evaluator from python2 and python3
```

Comment:
All of the features are available from python3 if one uses `python3 -m pip install --user --no-binary=correctionlib correctionlib`, only py2 is missing things other than the pybind11 interface

Reply:
Thanks, I added that option.